.travis.yml

```diff
@@ -50,7 +50,14 @@ matrix:
         # Cleanup the cached directories to avoid unnecessary cache updates
         - find $HOME/.ivy2/cache -name "ivydata-*.properties" -print -delete
         - find $HOME/.sbt        -name "*.lock"               -print -delete
-      script: npm install -g lerna && npm run node-ci
+      script: 
+        - npm install -g lerna
+        - npm run in-submodules -- -f categories.api=true -f language=typescript -- bootstrap --include-filtered-dependencies
```

Comment:
Should we also run `build` in each of the modules? So any possible errors during build process will be discovered?

Reply:
Yeah good point, done :).